server/publications/collections/media.js

```diff
@@ -1,80 +0,0 @@
-import { Meteor } from "meteor/meteor";
-import { check, Match } from "meteor/check";
-import { Media, Revisions } from "/lib/collections";
-import { Reaction } from "/server/api";
-import { RevisionApi } from "/imports/plugins/core/revisions/lib/api/revisions";
-
-
-/**
- * CollectionFS - Image/Video Publication
- * @params {Array} shops - array of current shop object
- */
-Meteor.publish("Media", function (shops) {
```

Comment:
As @mikemurray notes we still need `brandAssets` but I wonder if we still need all this code which seems like it just gets negated by 
```
return Media.find({
 -    "metadata.type": "brandAsset"
 -  });
```
at the bottom?

Reply:
@zenweasel @mikemurray It does seem that this publication could be simplified significantly.